test/studies/shootout/fasta/caseyb/fasta.chpl

```diff
@@ -115,16 +115,16 @@ proc randomMake(desc : string, a :[?D], n : int) {
 proc repeatMake(desc : string, alu : string, n : int) {
   stdout.write(desc);
   var r : int = alu.size;
-  var s : string = alu + alu + alu[1..n%r];
+  var s : string = alu + alu + alu[0..(n%r-1)];
```

Comment:
% takes precedence, right?  I think this is a little confusing to read regardless

Reply:
Ditto the fasta-lines version

Reply:
What change are you suggesting?

Reply:
An additional set of parentheses around `n%r`, I think.  Unless that really looks cluttered, I suppose, in which case I'd probably save the result of `n%r` into an intermediate variable and have this line use that and the `-1`